chapter03/Task01.py

```diff
@@ -29,10 +29,12 @@
     guess = int(input("Take a guess: "))
     tries += 1
     if tries == 10:
-        print('Ты не справился со мной, машины победили')
+        print('You lose! Machines WON!!! HA-HA-HA! ')
         break
+    elif guess != the_number:
+        continue
     else:
-        print("You guessed it!  The number was", the_number)
+        print('What?! You win?! How?!')
```

Comment:
you probably need a space here so that when the player wins, the "and it only took you..." isn't concatenated with that message